app/src/main/java/com/babylon/wallet/android/presentation/settings/securitycenter/backup/BackupViewModel.kt

```diff
@@ -77,8 +79,16 @@ class BackupViewModel @Inject constructor(
             Timber.tag("CloudBackup").d("Cloud backup authorization is in progress...")
             _state.update { it.copy(isCloudAuthorizationInProgress = true) }
             sendEvent(Event.SignInToGoogle)
-        } else { // just turn off the cloud backup sync
+        } else {
+            _state.update { it.copy(isCloudAuthorizationInProgress = true) }
+            ensureBabylonFactorSourceExistUseCase()
```

Comment:
What does this have to do with backups?

Reply:
you are right. I added the getprofileusecase